### PR TITLE
Report Garbage Collection & Runtime stats to prometheus

### PR DESCRIPTION
**Pull request by ChrisPenner**
I missed this the first time around because I didn't know what all these
stats were for; now that I'm going through the grafana dashboards I know what
they're for and it's worth porting over :)

### CODE REVIEW

File: libs/imports/src/Imports.hs [hunk 1]
```diff
@@ -4,7 +4,7 @@ module Imports
     -- * Base
       module Prelude
     , module Control.Applicative
-    , module Control.Monad
+    , module Control.Monad, whenM, unlessM
```
Comment (fisx) [inline libs/imports/src/Imports.hs]: Great, thanks for that!  Any reason why you didn't want to re-export the module?  (That would just re-export what's imported, no?)

Comment (neongreen) [reply libs/imports/src/Imports.hs]: Those functions are not in `base`, please move them elsewhere in the export list.

Comment (neongreen) [reply libs/imports/src/Imports.hs]: Also, thanks for adding them 🙂

Comment (ChrisPenner) [reply libs/imports/src/Imports.hs]: Ahh; didn't see the heading; just put them alongside the spot where I'd want them to be. I can move them.

**Comment by ChrisPenner** [comment]
@fisx I can't say about the threading issue; I notice that `galley` has `-N1` as an RTS option so I suspect it's still an issue; but note that OS Threads != Haskell Green Threads

At any rate, this doesn't change those settings so it should be fine 😄

**Pull request merged**